### PR TITLE
Update control.rb

### PR DESCRIPTION
**Pull request by everton-cerqueira**

### CODE REVIEW

**Comment by doudou** [comment]
Thanks !

**Comment by doudou** [comment]
Thanks again for the contribution, I appreciate it a lot.

However, just to be sure ... The files in `blueprints/` (more accurately, all the files that contain these backward-compatible-naming guards) should never be used in new bundles. They are really meant to be used by old bundles, as a backward compatibility measure. Older integrations are still using the `bundles/rock` package, however, which is why there are trivial mistakes in this package.

In summary: don't consider using those in new bundles.
